docs/Datagrid.md

```diff
@@ -896,6 +896,22 @@ const PostList = () => (
 );
 ```
 
+If you include a `<SelectColumnsButton>` in a page that has more than one `<DatagridConfigurable>`, you have to link the two components by giving them the same preferenceKey:
+
+```tsx
+const PostList = () => (
+    <List>
+        <SelectColumnsButton preferenceKey="posts.datagrid" />
+        <DatagridConfigurable preferenceKey="posts.datagrid">
+            <TextField source="id" />
+            <TextField source="title" />
+            <TextField source="author" />
+            <TextField source="year" />
+        </DatagridConfigurable>
+    </List>
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏This code snippet does not show a good example of how you should use `SelectColumnsButton` (it should be used inside the List `actions`). Please create an example based on [this one](https://marmelab.com/react-admin/SelectColumnsButton.html#usage) instead.